### PR TITLE
Settings for weekplanner feature 857

### PR DESCRIPTION
**Pull request by Taoshix**
# Description

Added Number of Actitivies to Display, Show Only Actitivies and Show Settings for Citizens to the web api and the settings table in the database to be used in the weekplanner and api client

Fixes related to #857 in weekplanner

## Type of change
*Delete unchecked boxes (only for Type of change)*

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

This has been tested by having the api client and weekplaner run tests that make requests to the web api to both get and insert data into the database. Specifically inputted and extracted data regarding the newly created attributes.

**Development Configuration**
*Type "flutter --version" and "dart --version" in your CMD to check versions.*

* Flutter version: 2.0.5
* Dart version: 2.12.3

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas, if necessary
- [ ] I have made corresponding changes to the documentation, if necessary
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works, if necessary
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have Acceptance Tested this on an iOS device
- [x] I have Acceptance Tested this on an Android device

### CODE REVIEW

File: GirafRest/Migrations/20221101082038_AddedShowOnlyActivitiesAndNrOfActivitiesToDisplayToSettingsTable.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
```
Comment (NikolajKrogh) [inline GirafRest/Migrations/20221101082038_AddedShowOnlyActivitiesAndNrOfActivitiesToDisplayToSettingsTable.cs]: Hvorfor har i tilføjet #nullable disable?

**Review by Jaffenheimer — CHANGES_REQUESTED** [review]
AddedShowOnlyActivitiesAndNrOfActivitiesToDisplayToSettingsTable.Designer.cs, AddedShowOnlyActivitiesAndNrOfActivitiesToDisplayToSettingsTable.cs og AddedShowSettingsForCitizenToSettingsTable.Designer.cs er efter min mening alt alt alt for lange klasse navne og bør ændres til noget mere kort og præcist. 

Resterende kommentarer til specifik kode kan ses i @NikolajKrogh's review.

**Comment by Jaffenheimer** [comment]
Har i testet dette også? Synes ikke jeg kan se at i har tilføjet nogle tests, bliver det i har lavet om allerede testet i forvejen?

File: GirafRest/Migrations/20221101082038_AddedShowOnlyActivitiesAndNrOfActivitiesToDisplayToSettingsTable.cs (hunk 1 above)
Comment (Taoshix) [reply GirafRest/Migrations/20221101082038_AddedShowOnlyActivitiesAndNrOfActivitiesToDisplayToSettingsTable.cs]: Dette er auto generated fra migration tool

**Review by Jaffenheimer — APPROVED** [review]
This looks good to me now

File: GirafRest/appsettings.Development.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿{
     "ConnectionStrings": {
-        "DefaultConnection": "server=192.38.56.153;port=3306;userid=root;password=Giraf123;database=giraf;Allow User Variables=True"
+        "DefaultConnection": "server=localhost;port=3306;userid=root;password=Giraf123;database=giraf;Allow User Variables=True"
```
Comment (AOsnes) [inline GirafRest/appsettings.Development.json]: Remove localhost from commmit

**Review by AOsnes — CHANGES_REQUESTED** [review]
Looks good except for the comment left

File: GirafRest/appsettings.Development.json (hunk 2 above)
Comment (Taoshix) [reply GirafRest/appsettings.Development.json]: Removed localhost now

**Comment by AOsnes** [comment]
TestUserCanCheckMultipleSettings integration test fails when running locally.

**Comment by Taoshix** [comment]
> TestUserCanCheckMultipleSettings integration test fails when running locally.

drop the giraf database then run `dotnet-ef database update`

There was a merge conflict in the last commit

**Review by AOsnes — APPROVED** [review]
lgtm

**Pull request merged**